roc/receiver_test.go

```diff
@@ -35,6 +35,46 @@ func TestReceiver_Open(t *testing.T) {
 			config:  ReceiverConfig{},
 			wantErr: newNativeErr("roc_receiver_open()", -1),
 		},
+		{
+			name: "invalid config.TargetLatency",
+			contextFunc: func() *Context {
+				ctx, err := OpenContext(makeContextConfig())
+				require.NoError(t, err)
+				return ctx
+			},
+			config:  makeReceiverConfig(WithTargetLatency(-1)),
```

Comment:
Regarding this bit:

```go
makeReceiverConfig(WithTargetLatency(-1))
```

I don't like a few things about this approach:

* test table become less self-contained; we need to look into implementation of functions like WithTargetLatency to see what they're doing

* we need to craft dozens of small functions, one per each field

* we complicate config construction by introducing concept of "options"

It seems there is a way to make things more simple and compact by modifying this approach a bit:

```go
name: "invalid config.TargetLatency",
contextFunc: func() *Context {
	ctx, err := OpenContext(makeContextConfig())
	require.NoError(t, err)
	return ctx
},
configFunc(): func() ReceiverConfig {
	cfg := makeReceiverConfig()
	cfg.TargetLatency = -1
	return cfg
},
```

Reply:
got the point